src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt

```diff
@@ -73,9 +73,12 @@ val RsFunction.isVariadic: Boolean
 val RsFunction.abiName: String?
     get() {
         val stub = greenStub
-        return stub?.abiName ?: abi?.litExpr?.stringValue
+        return stub?.abiName ?: abi?.litExpr?.stringValue ?: abi?.let { "C" }
```

Comment:
Some tests that render ABI in documentation popups failed, so I suppose we should provide two different properties:
- `literalAbiName` - that returns the ABI name how it is written in the code
- `actualAbiName` - that returns "semantical" ABI, i.e. returns "C" if ABI string is omitted

So, basically, you can rename `abiName` to `literalAbiName` and implement `actualAbiName` like this:
```kotlin
val RsFunction.actualAbiName: String?
    get() = literalAbiName ?: "C"
```

Reply:
Btw I see a mistake in the old `abiName` implementation. Could you please replace it with this?
```
val RsFunction.abiName: String?
    get() {
        val stub = greenStub
        if (stub != null) {
            return stub.abiName
        }
        return abi?.litExpr?.stringValue
    }
```
(the old implementation executes `abi?.litExpr?.stringValue` if `stub.abiName` is `null` while it should just return null)